lib/jormungandr/test/integration/Cardano/Faucet.hs

```diff
@@ -29,12 +34,19 @@ initFaucet :: IO Faucet
 initFaucet = do
     Faucet <$> newMVar mnemonics
 
-block0H :: Hash "Genesis"
-block0H = Hash $ unsafeFromHex (T.encodeUtf8 block0HText)
+getBlock0H :: IO (Hash "Genesis")
+getBlock0H = do
+    Hash . unsafeFromHex . T.encodeUtf8 <$> getBlock0HText
+
+getBlock0HText :: IO Text
+getBlock0HText = do
+    Stdout hash <- runJcli
```

Comment:
Maybe `readProcess` would have been better here than to use `runJcli`…  to easily avoid the un-exhaustive cases 🤔